code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1305,8 +1305,13 @@
 			oldspecies.handle_dna(src, TRUE) // Remove any genes that belong to the old species
 
 		oldspecies.on_species_loss(src)
+	var/datum/species/new_race
+	if(ispath(new_species))
+		new_race = new new_species
+	else if(istype(new_species))
+		new_race = new_species

```

Comment:
I'd rather this not be added; I'd really rather force people to create new instances of a species, on site, than it be a reference.

Reply:
sure, i'll remove it then